packages/account/src/providers/provider.ts

```diff
@@ -906,6 +916,22 @@ Supported fuel-core version: ${supportedVersion}.`
     return new TransactionResponse(transactionRequest, this, abis, subscription);
   }
 
+  /**
+   * Submits a transaction to the chain and awaits its status response.
+   *
+   * @param transactionRequestLike - the request to submit.
+   * @param sendTransactionParams - The provider send transaction parameters (optional).
+   * @returns A promise that resolves to a settled transaction.
+   */
+  async sendTransactionAndAwaitStatus(
+    transactionRequestLike: TransactionRequestLike,
+    providerSendTxParams: ProviderSendTxParams = {}
+  ): Promise<TransactionResult<void>> {
```

Comment:
```suggestion
  ) {
```
IMO you can remove this, but I don't mind it staying as well.

Reply:
Personally, I have always liked seeing the full typed function signature.